components/Drawer.tsx

```diff
@@ -0,0 +1,99 @@
+import Drawer from '@mui/material/Drawer'
+import List from '@mui/material/List'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import { Box } from '@mui/system'
+import StarIcon from '@mui/icons-material/Star'
+import Image from 'next/image'
+import Collapse from '@mui/material/Collapse'
+import ExpandLess from '@mui/icons-material/ExpandLess'
+import ExpandMore from '@mui/icons-material/ExpandMore'
+import React from 'react'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import SettingsIcon from '@mui/icons-material/Settings'
+
+export default function NavigationDrawer() {
```

Comment:
In the actual file directory, this should be in a `components/NavigationDrawer/index.tsx` file. See [KKB's old repo](https://github.com/hack4impact-utk/keep-knox-beautiful/tree/develop/src/components) for reference.